src/lib.rs

```diff
@@ -99,6 +102,87 @@ where
     namespace: BTreeMap<String, AmlValue>,
 }
 
+/// This is the entry point of `acpi` if you have no information except that the machine is running
+/// BIOS and not UEFI. It maps the RSDP, works out what version of ACPI the hardware supports, and
+/// passes the physical address of the RSDT/XSDT to `parse_rsdt`.
+///
+/// # Unsafety
+///
+/// This function is unsafe because it may read from protected memory if the computer is using UEFI.
+/// Only use this function if you are sure the computer is using BIOS.
+pub unsafe fn search_for_rsdp_bios<H>(handler: &mut H) -> Result<(), AcpiError>
+where
+    H: AcpiHandler,
+{
+    let ebda_start = find_ebda_start(handler);
```

Comment:
Weren't we going to change this to `find_ebda` to find the entire address range and simplify the `areas` definition below?

Reply:
Oh cool I can do that too